app/models/post.rb

```diff
@@ -2,4 +2,18 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] Kindly remove the `foreign_key: 'author_id'` option for the `belongs_to` because it is not needed in the Post model.
```suggestion
  belongs_to :author, class_name: 'User'
```